### ISSUE TITLE
posibility of a 2.1.3 release with updated d3-interpolate 

### ISSUE THREAD

**Issue by paimon0715** [recharts/recharts#2632]
- [ √] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### What is expected?
no warning

### What is actually happening?
warning

| Environment | Info |
|---|---|
| Recharts | v2.1.2 |
| React | 0.65.0 |
| System | windows |
| Browser | chorem |

---
Hi, @arcthur, 
### Issue Description

   I noticed that a vulnerability is introduced in **_recharts@2.1.2_**:
     Vulnerability **SNYK-JS-D3COLOR-1076592**  affects package **_d3-color_** (versions:<3.0.0):  [https://snyk.io/vuln/SNYK-JS-D3COLOR-1076592](https://snyk.io/vuln/SNYK-JS-D3COLOR-1076592)
  The above vulnerable package is referenced by **_recharts@2.1.2_** via:
`recharts@2.1.2 ➔ d3-interpolate@2.0.1 ➔ d3-color@2.0.0`

Since **_recharts@2.1.2_** is referenced by **142** downstream projects (e.g., [@types/recharts 2.0.0](https://www.npmjs.com/package/@types/recharts) (latest version), [blueprint-hooks-ui 0.57.29](https://www.npmjs.com/package/blueprint-hooks-ui) (latest version), [q3-admin 1.36.12](https://www.npmjs.com/package/q3-admin) (latest version), [@cheapreats/react-ui 2.4.36](https://www.npmjs.com/package/@cheapreats/react-ui) (latest version), [labo-components 6.0.2-ctu](https://www.npmjs.com/package/labo-components) (latest version)), the vulnerability [**SNYK-JS-D3COLOR-1076592**](https://snyk.io/vuln/SNYK-JS-D3COLOR-1076592) can be propagated into these downstream projects and expose security threats to them via the following package dependency paths:
 (1)`@alephdata/react-ftm@2.5.1 ➔ recharts@2.1.2 ➔ d3-interpolate@2.0.1 ➔ d3-color@2.0.0`
(2)`@cheapreats/react-ui@2.4.36 ➔ recharts@2.1.2 ➔ d3-interpolate@2.0.1 ➔ d3-color@2.0.0`
  **......**

If **_recharts@2.1.\*_** removes the vulnerable package from the above version, then its fixed version can help downstream users decrease their pain.

**Given the large number of downstream users, could you help update your package to remove the vulnerability from recharts@2.1.2 ?**

### Fixing suggestions

In **_recharts@2.1.3_**, maybe you can kindly try to perform the following upgrade :
`d3-interpolate ^2.0.1 ➔ ^3.0.0`; 

**Note:**
_**d3-interpolate@3.0.0**(>=3.0.0) directly depends on d3-color@3.0.1 which has fixed the vulnerability [**SNYK-JS-D3COLOR-1076592**](https://snyk.io/vuln/SNYK-JS-D3COLOR-1076592)._

Thank you for your attention to this issue and welcome to share other ways to resolve the issue.

**Comment by superwesker1988** [comment]
Can someone look at this issue, please? I notice there is also an open PR so maybe it wouldn't be difficult to release the suggested change?

**Comment by ckifer** [comment]
Closing this in favor of https://github.com/recharts/recharts/issues/3012

https://github.com/recharts/recharts/pull/3120 should be merged soon
